src/IndexManager.php

```diff
@@ -239,6 +255,60 @@ private function setIndexIfMapping()
         $this->indexIfMapping = $mapping;
     }
 
+    /**
+     * For each chunk performs the provided operation.
+     *
+     * @param  \Doctrine\Common\Persistence\ObjectManager $objectManager
```

Comment:
Nitpick: this class name is already imported, so no need for an FQCN.

Reply:
Thanks for the feedback. This is a minor subject, but I prefer keeping the FQCN in the phpdocs.